backend/controllers/prisonerProfile/prisonerCoursesQualificationsDetails.ts

```diff
@@ -0,0 +1,49 @@
+import { app } from '../../config'
+
+import { formatName, putLastNameFirst } from '../../utils'
+
+export default ({ prisonApi, esweService }) =>
+  async (req, res) => {
+    const { offenderNo } = req.params
+
+    type PrisonerDetails = {
+      offenderNo: string
+      firstName: string
+      lastName: string
+      bookingId: number
+      bookingNo: string
+      rootOffenderId: number
+      dateOfBirth: string
+      activeFlag: boolean
+      agencyId: string
+      assignedLivingUnitId: number
+    }
+
+    type coursesAndQuals = {
+      enabled: boolean
+      content: curious.LearnerEducationFullDetails[]
+    }
+
+    if (!app.esweEnabled) {
+      return res.redirect(`/prisoner/${offenderNo}`)
+    }
+
+    try {
+      const [prisonerDetails, coursesAndQualifications]: [PrisonerDetails, coursesAndQuals] = await Promise.all([
+        prisonApi.getDetails(res.locals, offenderNo),
+        esweService.getLearnerEducationFullDetails(offenderNo),
+      ])
```

Comment:
I think we need `logErrorAndContinue` here instead of a `try {}` block as it prevents a failure in one of these calls from stopping the whole prisoner profile page from being shown.

Reply:
So this is actually a completely separate page, not part of the prisoner profile - I also did have the logErrorAndContinue there in a map but it causes an almighty typescript error

Reply:
ah yes - i remember now you come to mention it! it fine as is then